Calibration/HcalAlCaRecoProducers/test/AlCaDiJetsProducer_cfg.py

```diff
@@ -0,0 +1,35 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("MYDIJETS")
+
+process.load("Configuration.Geometry.GeometryIdeal_cff")
+process.load("Configuration.StandardSequences.MagneticField_cff")
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+from Configuration.AlCa.autoCond_condDBv2 import autoCond
+process.GlobalTag.globaltag=autoCond['run2_mc']
```

Comment:
@bsunanda 
Two fixes in these tests (for a later PR):

1. Use `process.load('Configuration.StandardSequences.GeometryRecoDB_cff')` as explained by Ianna in one of the latest meetings: you need Run2 reco geometry.
2. Use `process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')` in order to load conditions from the new backend.